src/utils/format/index.ts

```diff
@@ -61,3 +61,13 @@ export const toQueryString = (value: {
 
 export const localeCurrencyToNumber = (value: string) =>
   Number(value.replaceAll(',', ''))
+
+export const removeUndefinedAndNull = (obj: {
```

Comment:
```suggestion
export const removeNullish = (obj: {
```
Undefined와 Null을 칭하는 단어로 Nullish는 어떠신가요?

Reply:
[반영 완료](https://github.com/price-offer/offer-fe/pull/241/commits/4dc6b204eaad5c8f3b150fc45139a90e9d72c449#diff-3b45b44c425221472815f71549f6fc4495500cdc1f22677df418a44dcb46dded) !